Sources/Call.swift

```diff
@@ -44,5 +46,81 @@ open class Call {
 
         return rootedJson[rootNode]
     }
-
+    
+    private func insertParameters(request: URLRequest) -> URLRequest {
+        if (self.parameters == nil) {
```

Comment:
🎈Dit lijkt me mooier met een guard.

```
guard let parameters = parameters else {
         return request
}
```

Waarom ook de request returnen. UrlRequest is kan je muteren want is een class. Dus is passed by reference. Alleen zijn alle parameters let als je ze als parameter doorgeeft. Dat kan je veranderen door in de guard hierboven let te veranderen door var.